src/tape/ensemble.py

```diff
@@ -472,6 +472,58 @@ def persist(self, **kwargs):
         self.update_frame(self.object.persist(**kwargs))
         self.update_frame(self.source.persist(**kwargs))
 
+    def sample_objects(self, **kwargs):
```

Comment:
I realize I suggested the name `sample_objects` but it might be worth checking in with someone like @nevencaplar or @hombit for input on the naming. Some other ideas are `sample_lightcurves`, `sample_lc`, or maybe even back to just `sample`.  

Personally I feel that having the fact that we're sampling on a per-object/per-light curve basis in the name is helpful but there may be disagreement

Reply:
`sample`, `sample_objects` or `random_objects` are all fine

Reply:
Good call, I'll post something on slack

Reply:
Majority vote seems to suggest `sample`, will go with that